Kconfig.zephyr

```diff
@@ -608,6 +608,22 @@ config WARN_EXPERIMENTAL
 	  Print a warning when the Kconfig tree is parsed if any experimental
 	  features are enabled.
 
+config ENFORCE_ZEPHYR_STDINT
+	bool
+	prompt "Enforce Zephyr convention for stdint"
```

Comment:
late to the game, but should this really have a prompt ?

Wouldn't it be better to have this promptless so for example a pre-built library can makes a `Kconfig` file that will default this symbol to `n` when said library is selected and linked ?

I believe it would be for such libraries to know whether or not this should be enforced and not the enduser.